web/src/helpers/tooltip.js

```diff
@@ -174,37 +169,43 @@ module.exports.showExchange = function(tooltipInstance, key, country, displayByE
   tooltipInstance.show();
 };
 
-module.exports.showMapCountry = function(tooltipInstance, countryData, co2color, co2Colorbars) {
-  if (countryData.co2intensity && co2Colorbars)
+module.exports.showMapCountry = function showMapCountry(tooltipInstance, countryData, co2color, co2Colorbars) {
+  if (countryData.co2intensity && co2Colorbars) {
     co2Colorbars.forEach((c) => { c.currentMarker(countryData.co2intensity); });
+  }
   const tooltip = d3.select(tooltipInstance._selector);
   tooltip.select('#country-flag')
     .attr('src', flags.flagUri(countryData.countryCode, 16));
   tooltip.select('#country-name')
     .text(translation.getFullZoneName(countryData.countryCode))
     .style('font-weight', 'bold');
-  tooltip.select('.emission-rect')
-    .style('background-color', countryData.co2intensity ? co2color(countryData.co2intensity) : 'gray');
-  tooltip.select('.country-emission-intensity')
-    .text(Math.round(countryData.co2intensity) || '?');
-
-  const priceData = countryData.price || {};
-  const hasFossilFuelData = countryData.fossilFuelRatio !== null;
-  const fossilFuelPercent = countryData.fossilFuelRatio * 100;
-  tooltip.select('.lowcarbon-percentage')
-    .text(hasFossilFuelData ? Math.round(100 - fossilFuelPercent) : '?');
-  const hasRenewableData = countryData.renewableRatio !== null;
-  const renewablePercent = countryData.renewableRatio * 100;
-  tooltip.select('.renewable-percentage')
-    .text(hasRenewableData ? Math.round(renewablePercent) : '?');
+
+  if (countryData.hasParser) {
+    tooltip.select('.zone-details').style('display', 'block');
+    tooltip.select('.emission-rect')
+      .style('background-color', countryData.co2intensity ? co2color(countryData.co2intensity) : 'gray');
+    tooltip.select('.country-emission-intensity')
+      .text(Math.round(countryData.co2intensity) || '?');
+
+    const hasFossilFuelData = countryData.fossilFuelRatio !== null;
```

Comment:
!= null will check for undefined also, so it might be a good idea to remove one `=`